busylight/api/busylight_api.py

```diff
@@ -499,7 +542,8 @@ async def flash_light_impressively(
     rgb_a = parse_color_string(color_a, dim)
     rgb_b = parse_color_string(color_b, dim)
 
-    fli = Effects.for_name("blink")(rgb_a, speed.duty_cycle / 10, off_color=rgb_b)
+    fli = Effects.for_name("blink")(
+        rgb_a, speed.duty_cycle / 10, off_color=rgb_b)
```

Comment:
FYI It's generally poor etiquette to include formatting-only changes in a pull request. I'm not going to ask you update the PR just for this, but others might.  Besides, after I merge the PR I'm going to run `black` on everything to homogenize formatting.

Reply:
Thank you so much for the heads up! I didn't know that. I'll clean up my editor's attempts at being "helpful" and adhere to etiquette :)